packages/airnode-deployer/src/cli/commands.ts

```diff
@@ -96,14 +100,39 @@ export async function deploy(configPath: string, secretsPath: string, receiptFil
     throw goDeployAirnode.error;
   }
 
-  const output = goDeployAirnode.data;
-  const deploymentTimestamp = new Date().toISOString();
+  if (!goDeployAirnode.success) {
+    logger.fail(
+      bold(
+        `Airnode deployment failed due to unexpected errors.\n` +
+          `  It is possible that some resources have been deployed on cloud provider.\n` +
+          `  We are attempting to remove them...\n`
```

Comment:
I think we should strongly avoid "we" terminology in messages, or more broadly anything that implies the core team has some kind of admin ability to interfere with deployments. It's misleading and is likely going to cause confusion for users. 

I was looking at the screenshots and noticed this

Reply:
Would:

```
It is possible that some resources have been deployed on cloud provider.
Attempting to remove them...
```

be OK?

Reply:
Yeah that's better 👍🏻